crates/rune-modules/src/base64.rs

```diff
@@ -23,7 +23,7 @@ pub fn module(_stdio: bool) -> Result<Module, ContextError> {
 /// ```
 #[rune::function]
 fn decode(inp: &str) -> Result<Vec<u8>, String> {
-    Ok(BASE64_STANDARD.decode(inp).map_err(|e| e.to_string())?)
```

Comment:
Please define a wrapper around the base64 error typ8e, else the error type exported to Rune scripts will be strings rather than an appropriate error type.

The error type primarily needs to implement the `STRING_DISPLAY` protocol. You can see [an example here](https://github.com/rune-rs/rune/blob/52c52f5d93c2336aa4434cd713c619384b70a2c1/crates/rune-modules/src/http.rs#L96).

Reply:
Moving this conversation.

Reply:
Yeah good point. Will do it